docs/install.md

```diff
@@ -0,0 +1,204 @@
+# Installing Tekton Dashboard
+
+This guide explains how to install Tekton Dashboard. It covers the following topics:
+
+* [Before you begin](#before-you-begin)
+* [Pre-requisites](#pre-requisites)
+* [Which version should I use](#which-version-should-i-use)
+* [Installing Tekton Dashboard on Kubernetes](#installing-tekton-dashboard-on-kubernetes)
+* [Installing Tekton Dashboard on OpenShift](#installing-tekton-dashboard-on-openshift)
+* [Accessing the Dashboard on Kubernetes](#accessing-the-dashboard-on-kubernetes)
+* [Accessing the Dashboard on OpenShift](#accessing-the-dashboard-on-openshift)
+* [Uninstalling the Dashboard on Kubernetes](#uninstalling-the-dashboard-on-kubernetes)
+* [Next steps](#next-steps)
+
+## Before you begin
+
+1. Choose the version of Tekton Dashboard you want to install. You have the following options:
+
+   * **[Official](https://github.com/tektoncd/dashboard/releases)** - install this unless you have a specific reason to go for a different release.
+   * **[Nightly](../tekton/README.md#nightly-releases)** - may contain bugs, install at your own risk. Nightlies live at `gcr.io/tekton-nightly`.
+   * **[`HEAD`]** - this is the bleeding edge. It contains unreleased code that may result
+     in unpredictable behavior. To get started, see the [development guide](../DEVELOPMENT.md) instead of this page.
+
+2. If you don't have an existing Kubernetes cluster, set one up, version 1.15 or later:
+
+   ```bash
+   #Example command for creating a cluster on GKE
+   gcloud container clusters create $CLUSTER_NAME \
+     --zone=$CLUSTER_ZONE --cluster-version=1.15.11-gke.5
+   ```
+
+3. Grant `cluster-admin` permissions to the current user:
+
+   ```bash
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole=cluster-admin \
+   --user=$(gcloud config get-value core/account)
+   ```
+
+   See [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control)
+   for more information.
+
+## Pre-requisites
+
+In order to install the Tekton Dashboard, please make sure the following requirements are met:
+- A [kubernetes cluster](https://github.com/kubernetes/kubernetes) must be installed with version 1.15.0 or later.
+- [Tekton Pipelines](https://github.com/tektoncd/pipeline) must be installed in the cluster.
+  Instructions to install Tekton Pipelines can be found [here](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
```

Comment:
copied from https://github.com/tektoncd/dashboard/pull/1476#discussion_r436833891

Do we want to link to docs in other project repos or on the website?

e.g. for PIpelines install: https://tekton.dev/docs/getting-started/#installation

also avoid 'here', 'click here' etc. as the clickable text in links, instead prefer to use the target page title or similar to give context. This is especially important for users of screenreaders. Depending on how they're navigating the page they may not otherwise have enough context to know where the link goes.

Reply:
fixed, keeping github links for now, if possible i would like to stay on github when coming from github and stay on the website when coming from the website

Reply:
> i would like to stay on github when coming from github and stay on the website when coming from the website

This is going to be tricky 😆 let's ignore the website for now, there was a proposal opened last night that may change how we approach the docs entirely...